crates/ruff_linter/src/rules/flake8_pyi/rules/unnecessary_type_union.rs

```diff
@@ -144,24 +147,32 @@ pub(crate) fn unnecessary_type_union<'a>(checker: &mut Checker, union: &'a Expr)
                         range: TextRange::default(),
                     }))
```

Comment:
Do we not need to take the `other_exprs` into account in the top part of this `if`-`else`?

Reply:
I'll double-check!